<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RTools</class>
 <widget class="QDialog" name="RTools">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tools</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Actions</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QToolButton" name="btnMoveTable">
          <property name="text">
           <string>Move
table</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/images/transfer.png</normaloff>:/images/transfer.png</iconset>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QToolButton" name="btnOrderComment">
          <property name="text">
           <string>Order
comment</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/images/mark.png</normaloff>:/images/mark.png</iconset>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QToolButton" name="btnRemoveOrder">
          <property name="text">
           <string>Remove
order</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/images/garbage.png</normaloff>:/images/garbage.png</iconset>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QToolButton" name="btnChangePassword">
          <property name="text">
           <string>Change my
password</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/images/lock.png</normaloff>:/images/lock.png</iconset>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QToolButton" name="btnComplex">
          <property name="text">
           <string>Add complex</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/images/dinner.png</normaloff>:/images/dinner.png</iconset>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QToolButton" name="btnShowHideRemovedItems">
          <property name="text">
           <string>Show / hide
removed items</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/images/basket.png</normaloff>:/images/basket.png</iconset>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QToolButton" name="btnCompimentary">
          <property name="text">
           <string>Complimentary</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/images/gift.png</normaloff>:/images/gift.png</iconset>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QToolButton" name="btnTax">
          <property name="text">
           <string>Print tax</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/images/tax.png</normaloff>:/images/tax.png</iconset>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QToolButton" name="btnRecover">
          <property name="text">
           <string>Recover</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/images/heart-pulses.png</normaloff>:/images/heart-pulses.png</iconset>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QToolButton" name="btnPrintAnyReport">
          <property name="text">
           <string>Print receipt
by number</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/images/invoice.png</normaloff>:/images/invoice.png</iconset>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QToolButton" name="btnVoidBack">
          <property name="text">
           <string>Void Back</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/images/recycle.png</normaloff>:/images/recycle.png</iconset>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QToolButton" name="btnPrintVoid">
          <property name="text">
           <string>Print void
report</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/images/recycle.png</normaloff>:/images/recycle.png</iconset>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Reports</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="2">
         <widget class="QToolButton" name="btnExit">
          <property name="text">
           <string>Exit</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/images/exit.png</normaloff>:/images/exit.png</iconset>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QToolButton" name="btnShowTableOrders">
          <property name="text">
           <string>Show table
orders</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/images/invoice.png</normaloff>:/images/invoice.png</iconset>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QToolButton" name="btnTotalReport">
          <property name="text">
           <string>Total, Today</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/images/money_calculator.png</normaloff>:/images/money_calculator.png</iconset>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QToolButton" name="btnTotalYesterday">
          <property name="text">
           <string>Total, Yesterday</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/images/money_calculator.png</normaloff>:/images/money_calculator.png</iconset>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QToolButton" name="btnShowNyTotal">
          <property name="text">
           <string>Show my total</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/images/money_calculator.png</normaloff>:/images/money_calculator.png</iconset>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QToolButton" name="btnTotalReportAnyDay">
          <property name="text">
           <string>Total, Any day</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/images/money_calculator.png</normaloff>:/images/money_calculator.png</iconset>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
